lxd/container.go

```diff
@@ -42,12 +42,12 @@ func init() {
 			return nil, err
 		}
 
-		instances := []device.InstanceIdentifier{}
+		identifiers := []device.InstanceIdentifier{}
 		for _, v := range containers {
-			instances = append(instances, device.InstanceIdentifier(v))
+			identifiers = append(identifiers, device.InstanceIdentifier(v))
 		}
 
-		return instances, nil
+		return identifiers, nil
```

Comment:
This part of the change was probably meant to get merged with https://github.com/lxc/lxd/pull/5900/commits/95979122cb37785b0401d9aad0e08e0c8eaf3630 instead